core/src/main/resources/hudson/cli/CLIAction/example.jelly

```diff
@@ -34,6 +34,6 @@ THE SOFTWARE.
       }
     </style>
   </st:once>
-  <pre id="example">java -jar <a href="${rootURL}/jnlpJars/jenkins-cli.jar" style="color: white">jenkins-cli.jar</a> -s ${h.inferHudsonURL(request)} <j:if test="${it.webSocketSupported}">-webSocket</j:if> ${commandArgs}</pre>
+  <pre id="example">java -jar <a href="${rootURL}/jnlpJars/jenkins-cli.jar" style="color: white">jenkins-cli.jar</a> -s ${h.jnlpRootUrl(request)} <j:if test="${it.webSocketSupported}">-webSocket</j:if> ${commandArgs}</pre>
```

Comment:
The URL used for the CLI, which uses HTTP, should not generally be changed. Anyway this is only a visual suggestion which you are free to adjust or ignore.